app/src/main/java/com/eveningoutpost/dexdrip/Models/TransmitterData.java

```diff
@@ -58,7 +58,7 @@ public static TransmitterData create(byte[] buffer, int len, Long timestamp) {
 
             randomDelay(100, 1000);
             TransmitterData lastTransmitterData = TransmitterData.last();
-            if (lastTransmitterData != null && lastTransmitterData.raw_data == Integer.parseInt(data[0]) && Math.abs(lastTransmitterData.timestamp - timestamp) < (10000)) { //Stop allowing duplicate data, its bad!
+            if (lastTransmitterData != null && lastTransmitterData.raw_data == Integer.parseInt(data[0]) && Math.abs(lastTransmitterData.timestamp - timestamp) < (100000)) { //Stop allowing duplicate data, its bad! // jamorham: increased to 100 seconds for additional safety with multiple collection modes operating at the same time
                 return null;
             }
             if (data.length > 1) { transmitterData.sensor_battery_level = Integer.parseInt(data[1]); }
```

Comment:
stupid thing, does not let me put the comment also from below.
need to increase the 10000, to 10000 also on line 77.
I would actually make this 120,000 to be more consistent.